### PR TITLE
IARCRP-18: Open Lektor edit links in a new tab

### PR DESCRIPTION
**Pull request by mjecov**

### CODE REVIEW

File: src/lektorium/client/src/components/ControlPanel.vue [hunk 1]
```diff
@@ -29,7 +29,8 @@
               <td>
                 <a
                   v-if="site.productionUrl.startsWith('http')"
-                  :href="site.productionUrl"
+                  href="#"
+                  @click="openNewTab(site.productionUrl)"
```
Comment (apatrushev) [inline src/lektorium/client/src/components/ControlPanel.vue]: Why not?
```
target="_blank"
```

Comment (mjecov) [reply src/lektorium/client/src/components/ControlPanel.vue]: I thought it was unsafe to use "_blank"

Comment (apatrushev) [reply src/lektorium/client/src/components/ControlPanel.vue]: Any links to describe safety problem?

Comment (mjecov) [reply src/lektorium/client/src/components/ControlPanel.vue]: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Comment (apatrushev) [reply src/lektorium/client/src/components/ControlPanel.vue]: > https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Have you read the article?

Comment (mjecov) [reply src/lektorium/client/src/components/ControlPanel.vue]: Yes, it looks like we don`t need it in our case. Reinsured, will fix it. Thank you!

**Pull request merged**